.github/workflows/files-presence-validation.yml

```diff
@@ -0,0 +1,12 @@
+name: Validate File Presence
+on:
+  push:
+    branches:
+      - redat97/issue14
+  workflow_dispatch:
+
+jobs:
+  file-validation:
+   uses: ai-cfia/github-workflows/.github/workflows/workflow-check-file-presence.yml@redat97/issue8
+   with:
+    files: 'test_documentation.md, test_file.md'
```

Comment:
TESTING.md, README.md, LICENSE.md

I don't think it's worth creating a generic file checker for which we have to specify the list of files. If you have an additional workflow, create instead repository-validation.yml that includes among other things checking for the presence of TESTING.md, README.md, LICENSE.md

Reply:
ah this was just an example to test, not the actual files that need to be checked. I thought different repos could have certain mandatory files that are different but I can make a generic one for sure

Reply:
sure, but then the parameter could be an higher level project-type: Python. Doing it this way clobbers the usefulness and doesn't really express what we actually want.